list-doi-requests/src/main/java/no/unit/nva/doirequest/list/ListDoiRequestsHandler.java

```diff
@@ -0,0 +1,93 @@
+package no.unit.nva.doirequest.list;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+import no.unit.nva.model.Publication;
+import no.unit.nva.publication.service.impl.DoiRequestService;
+import no.unit.nva.publication.storage.model.DoiRequest;
+import no.unit.nva.publication.storage.model.UserInstance;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.core.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ListDoiRequestsHandler extends ApiGatewayHandler<Void, Publication[]> {
+
+    public static final String ROLE_QUERY_PARAMETER = "role";
+    public static final String CURATOR_ROLE = "Curator";
```

Comment:
I suppose in reality, these should be in param store, but this is an improvement, not aa necessary change